src/Tickets/Commerce/Gateways/Square/Syncs/Remote_Objects.php

```diff
@@ -352,11 +352,11 @@ public function get_line_item( array $item, WP_Post $order ): array {
 		}
 
 		$remote_object_id = Item::get_remote_object_id( $ticket_id );
+		$event_id         = $ticket_object->get_event()->ID;
 
 		$updates = [
-			'name'             => $ticket_object->get_event()->post_title,
+			'uid'              => "e-{$event_id}-t-{$ticket_id}",
```

Comment:
This is read only property -we cant specify it it will break the sync.

Also why are you specifying the names only when you have a remote object id ?